src/main/java/com/mixpanel/android/mpmetrics/Tweaks.java

```diff
@@ -58,8 +58,9 @@ public synchronized boolean isNewValue(String tweakName, Object value) {
             MPLog.w(LOGTAG, "Attempt to reference a tweak \"" + tweakName + "\" which has never been defined.");
             return false;
         }
-
         final TweakValue container = mTweakValues.get(tweakName);
+        if (container == null || container.value == null)
```

Comment:
We check for `containsKey()` in line 57 so `container == null` is not needed here.
Also, we avoid one-liner if statements without `{}`